extensions/ao-raycast/package.json

```
@@ -0,0 +1,128 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "ao-raycast",
+  "type": "module",
+  "title": "Ao Raycast",
+  "description": "Search and browse ArNS names with gateway support",
+  "icon": "ao-512.png",
+  "author": "dpshade22",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "arns",
+      "title": "Search Arweave Domains",
+      "label": "Search Arweave Domains",
+      "description": "Search and browse ArNS names with gateway support",
+      "mode": "view"
+    },
+    {
+      "name": "openData",
+      "title": "Open Transaction Data",
+      "label": "Open Transaction Data",
+      "description": "View raw data of an Arweave transaction using configurable gateways",
+      "mode": "no-view",
+      "arguments": [
+        {
+          "name": "txId",
+          "type": "text",
+          "placeholder": "Arweave Transaction ID",
+          "required": false,
+          "validation": "^.{43}$"
+        }
+      ]
+    },
+    {
+      "name": "openExplorer",
+      "title": "Open in Explorer",
+      "label": "Open in Explorer",
```

```suggestion
      "subtitle": "Ao Raycast",
```

No such thing called label in the API